src/pybind/rbd/rbd.pyx

```diff
@@ -4094,7 +4094,7 @@ written." % (self.name, ret, length))
             ret = rbd_get_access_timestamp(self.image, &timestamp)
         if ret != 0:
             raise make_ex(ret, 'error getting access timestamp for image: %s' % (self.name))
-        return datetime.fromtimestamp(timestamp.tv_sec)
+        return datetime.utcfromtimestamp(timestamp.tv_sec)
```

Comment:
`datetime.fromtimestamp(timestamp.tv_sec, timezone.utc)` is preferred. `utcfromtimestamp` is deprecated from `3.12`
See https://docs.python.org/3/library/datetime.html#datetime.datetime.utcfromtimestamp.

We may want to replace all other `utcfromtimestamp()` calls in the library

Reply:
The result produced by these calls is not identical.  The former is a `datetime` object with with `tzinfo=None`:
```
>>> datetime.datetime.utcfromtimestamp(123456)
datetime.datetime(1970, 1, 2, 10, 17, 36)
>>> datetime.datetime.fromtimestamp(123456, datetime.timezone.utc)
datetime.datetime(1970, 1, 2, 10, 17, 36, tzinfo=datetime.timezone.utc)
```
Given that this is intended to be backported, I'd rather align to the existing usage.  The replacement would need to concern all of `src/pybind/rbd/rbd.pyx` and be main-only.

Reply:
Ah okay!